app/models/spree/address.rb

```diff
@@ -4,7 +4,7 @@ module Spree
   class Address < ApplicationRecord
     include AddressDisplay
 
-    searchable_attributes :firstname, :lastname
+    searchable_attributes :firstname, :lastname, :phone
```

Comment:
The address model has another attribute named `alternative_phone` 
Should we use it too?

Reply:
As far as I know and as I read, this `alternative_phone` is not widely used ;) 
Maybe we should delete it?